library/alloc/src/borrow.rs

```diff
@@ -103,6 +103,11 @@ where
 /// is desired, `to_mut` will obtain a mutable reference to an owned
 /// value, cloning if necessary.
 ///
+/// If you need reference-counting pointers, note that
```

Comment:
Maybe better something like `If you need shared reference-counted access to ...` instead of just `pointers`?

Reply:
I was using `pointer` since `Cow` calls itself a “smart pointer” and `Rc` and `Arc` describe themselves as “reference-counting pointer[s]”. Furthermore, `Cow` already provides shared access, so the word “shared” is not really differentiating `Rc`/`Arc` from `Cow`. The thing that reference-counting provides is shared ownership, so “access” might not be the best word either; furthermore I don’t think there should be any further explanation of what reference-counting pointers are/do on a page about the `Cow` enum; the main point of this new paragraph is the fact that `Rc::make_mut` and `Arc::make_mut` are mentioned (and linked) at all.

But if you’re unhappy with my phrasing, feel free to give some more explanation on why you’d prefer a different phrasing 😉; the current one is not set in stone. E.g. one could discuss dropping the word “pointers” entirely or using a singular form.